modules/ml/src/main/java/org/apache/ignite/ml/math/VectorUtils.java

```diff
@@ -39,4 +39,62 @@ public static DenseLocalOnHeapVector zeroes(int n) {
     public static Vector fromMap(Map<Integer, Double> val, boolean cp) {
         return new MapWrapperVector(val);
     }
+
+    /**
+     * Turn number into Vector of given size with one-hot encoding.
+     *
+     * @param num Number to turn into vector.
+     * @param vecSize Vector size of output vector.
+     * @return One-hot encoded number.
+     */
+    public static Vector num2Vec(int num, int vecSize) {
```

Comment:
add isDistributed flag and produce different types of vector, not only local dense

Reply:
Done.